sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_create_test_live.cpp

```diff
@@ -1,15 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "gtest/gtest.h"
-
+#include "../../azure-security-attestation/src/private/crypto/inc/crypto.hpp"
```

Comment:
We should make a copy/paste of the crypto components instead of depending on attestation, IMHO.

Reply:
i'd rather include the file as it is bound to change due to that fact that it's not yet released